BMM.UI.Android/Properties/AndroidManifest.xml

```diff
@@ -31,6 +31,7 @@
 	</application>
 	<uses-feature android:name="android.hardware.output" android:required="false" />
 	<permission android:name="org.brunstad.bmm.permission.C2D_MESSAGE" android:protectionLevel="signature" />
+	<uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
```

Comment:
does this mean that with the next update ~40% of our users will see a message, asking to give the notification permission?
(That's not a bad thing. Just making sure I understand)

Reply:
No, it's only for new installations on Android 13 or never OS.
If you open the app on older OS, notification permission is always granted automatically.
If you use the app on Android 12 and then you update the OS to Android 13 you won't see the message too.